### PR TITLE
🎨  Avoid template syntax when not needed

### PR DESCRIPTION
**Pull request by amortemousque**
## Motivation

Avoid template syntax when not needed
Avoid useless concat (no issue related in the code but useful for future development)

## Changes

Update eslintrc

## Testing

- [ ] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -64,11 +64,13 @@ module.exports = {
     'no-template-curly-in-string': 'error',
     'no-throw-literal': 'error',
     'no-undef-init': 'error',
+    'no-useless-concat': 'error',
```
Comment (amortemousque) [inline .eslintrc.js:67]: no issue related in the code but useful for future development

**Review by BenoitZugmeyer — APPROVED** [review]
Nice!

File: scripts/cli [hunk 2]
```diff
@@ -28,7 +28,7 @@ cmd_typecheck () {
 
 cmd_lint () {
   local project_path="${1}"
-  eslint "$project_path"
+  eslint "$project_path"
```
Comment (bcaudan) [inline scripts/cli:31]: **❓ question:** ‏not sure to see what is going on here 🤔 involuntary update?

**Review by bcaudan — APPROVED** [review]

File: scripts/cli (hunk 2 above)
Comment (amortemousque) [reply scripts/cli:31]: Nice catch.
Yes it's an involuntary update from when I set `--fix`

**Pull request merged**